unattended-upgrade

```diff
@@ -806,6 +832,14 @@ def check_changes_for_sanity(cache, allowed_origins, blacklist, whitelist,
     return True
 
 
+def is_deb(file):
+    # type: (str) -> bool
+    if re.match("^.*\\.deb$", file):
```

Comment:
You could use `file.endswith(".deb")` here, no?

Reply:
Indeed, fixed.